src/build.d

```diff
@@ -950,7 +958,7 @@ struct Dependency
             command[i] = c.replace("$@", target);
 
         // Support $< (shortcut for the source path)
-        if (command[$ - 1].find("$<"))
+        if (command[$ - 1] == "$<")
```

Comment:
Isn't a different PR fixing this too?

Reply:
Yes, but I needed the fix for this one to work, so I temporarily keep it here until the other PR gets merged.